shell/platform/windows/client_wrapper/include/flutter/flutter_view.h

```diff
@@ -9,6 +9,9 @@
 
 namespace flutter {
 
+// The unique identifier for a view.
+typedef int64_t FlutterViewId;
```

Comment:
This `FlutterViewId` typedef is new, the original design used `int64_t` directly.